packages/framework/fluid-static/src/types.ts

```diff
@@ -69,6 +69,43 @@ export interface ContainerSchema {
  * Events that trigger when the roster of members in the Fluid session change.
  * Only changes that would be reflected in the returned map of IServiceAudience's getMembers method
  * will emit events.
+ *
+ * ### "membersChanged"
+ *
+ * The membersChanged event is emitted when a member is either added or removed.
+ *
+ * #### Listener signature
+ *
+ * ```typescript
+ * () => void;
+ * ```
+ *
+ * ### "memberAdded"
+ *
+ * The memberAdded event is emitted when a member joins the audience.
+ *
+ * #### Listener signature
+ *
+ * ```typescript
+ * (clientId: string, member: M) => void;
+ * ```
+ * - `clientId` - A unique identifier for the client
+ *
+ * - `member` - The service-specific member object for the client
+ *
+ * ### "memberRemoved"
+ *
+ * The memberRemoved event is emitted when a member leaves the audience.
+ *
+ * #### Listener signature
+ *
+ * ```typescript
+ * (clientId: string, member: M) => void;
+ * ```
+ * - `clientId` - A unique identifier for the client
+ *
+ * - `member` - The service-specific member object for the client
+ * @typeParam M - A service-specific member type.
```

Comment:
This doesn't actually produce any output in the documentation -- might be another bug to investigate.

Reply:
for anyone wondering, it's an open issue on their end: https://github.com/microsoft/rushstack/issues/1635